mmv1/products/compute/BackendService.yaml

```
@@ -665,6 +691,16 @@ properties:
               required: true
               description: |
                 The header field name to match on when bypassing cache. Values are case-insensitive.
+      - !ruby/object:Api::Type::Boolean
+        name: requestCoalescing
+        description: |-
+          If true then Cloud CDN will combine multiple concurrent cache fill
+          requests into a small number of requests to the origin.
+      - !ruby/object:Api::Type::Array
+        name: signedUrlKeyNames
+        description: '[Output Only] Names of the keys for signing request URLs.'
```

You can remove [Output Only] here, this will be shown in documentation as an attribute which doesn't accept user input
```suggestion
        description: 'Names of the keys for signing request URLs.'
```